test/unit/iterator_type_test.lua

```diff
@@ -0,0 +1,29 @@
+local expirationd = require('expirationd')
+local t = require('luatest')
+local g = t.group("iterator_type")
```

Comment:
nit: Above you use `'` but here you use `"`. I think it's better to keep the same "style" for string definition.

Reply:
Fixed for all commits and for expirationd legacy code too